content/blog/2017/2017-01-12-scm-api-2.0-release.adoc

```diff
@@ -0,0 +1,192 @@
+---
+:layout: post
+:title: SCM API turns 2.0 and what that means for you
+:tags:
+- development
+	- plugins
+:author: stephenc
+---
+
+=== The back-story
```

Comment:
This reads more like a History of Literate plugin than an advisory about potential upgrade problems.

I think we should start with something like "We've made some great improvements to the GitHub Branch Source & BitBucket branch source plugins, but there are some gotcha's that Jenkins administrators will need to be aware of. 

We want to give you the whole story, but the TL;DR; is: When updating your plugins, please be sure to upgrade the GitHub Branch Source and/or Bitbucket branch source plugins. Doing partial upgrades is bad news!"